# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux-header/kdev_t.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Fixed the direct inclusion of kdev_t.h e.g. from userspace apps.

  - Rene Rebe <rene@exactcode.de>

--- linux-2.6.8.1/include/linux/kdev_t.h.orig	2004-09-29 10:42:47.335008887 +0200
+++ linux-2.6.8.1/include/linux/kdev_t.h	2004-09-29 10:43:08.218675269 +0200
@@ -1,5 +1,8 @@
 #ifndef _LINUX_KDEV_T_H
 #define _LINUX_KDEV_T_H
+
+#include<linux/types.h>
+
 #ifdef __KERNEL__
 #define MINORBITS	20
 #define MINORMASK	((1U << MINORBITS) - 1)
